File: python/days_2_timestamps.py
from math import ceil


def days_2_timestamps(days):
    """
    convert the days to #timestamps
    NOTE Not work for array

    Liyan Song Nov. 2019
    """

    timestamps_in_1day = 86400
    timestamps_of_days = ceil(days * timestamps_in_1day)
    return timestamps_of_days

File: matlab/for_RQs/days2timestamps.m
function timestamps = days2timestamps(days)
% old name: cvtDay2_tmstamp
% Convert days to Unix timestamp. 
% 
% Liyan Song on 2019/10/30
% 

CONFIG = configure();
ONEDAYSTAMP = CONFIG.ONEDAYSTAMP;

timestamps = ceil(days * ONEDAYSTAMP);
end